TAO/tests/Bug_3251_Regression/DllOrb.h

```diff
@@ -1,34 +1,34 @@
 #ifndef bug_3251_DllORB_h
 #define bug_3251_DllORB_h
 
-#include "ace/Auto_Ptr.h"
 #include "ace/Barrier.h"
 #include "ace/Task.h"
 #include "tao/ORB.h"
 
-#include "tao/PortableServer/PortableServer.h"
 #include "bug_3251_export.h"
+#include "tao/PortableServer/PortableServer.h"
 
+#include <memory>
 
-class bug_3251_Export DllOrb: public ACE_Task_Base
+class bug_3251_Export DllOrb : public ACE_Task_Base
 {
 public:
   DllOrb (void);
   virtual ~DllOrb (void);
 
-  CORBA::ORB_ptr orb () const { return CORBA::ORB::_duplicate(mv_orb_.in()); }
+  CORBA::ORB_ptr orb () const { return CORBA::ORB::_duplicate (mv_orb_.in ()); }
 
-  virtual int init (int argc, ACE_TCHAR *argv[]);
+  virtual int init (int argc, ACE_TCHAR* argv[]);
 
   virtual int fini ();
 
   virtual int svc ();
 
 private:
-  auto_ptr < ACE_Thread_Barrier >      ma_barrier_;
-  CORBA::ORB_var                       mv_orb_;
-  PortableServer::POA_var              mv_rootPOA_;
-  PortableServer::POAManager_var       mv_poaManager_;
+  std::unique_ptr<ACE_Thread_Barrier> ma_barrier_;
```

Comment:
In general we do not align the members, these test had that but now we define a generic clang-format for ACE/TAO I would prefer to update these tests to follow the guidelines

Reply:
OK, I will change it